README.md

```diff
@@ -23,4 +23,6 @@ The parent chooses which child these vaccines were for, and the details of the v
 - NGO who provided vaccine
 - Date vaccine given
 
-
+## Slack Channel
+This is the slack channel for those who would like to communicate and collaborate with team members via slack.
+Once you have joined EmpowerHack_LDN, the channel for VaccinePass is healthldn_vaccinepass.
```

Comment:
Looks good 👍 . I would change the formatting slightly and put the channel in back ticks to stand out more.

--- 
For example...

## Slack Channel
 
This is the slack channel for those who would like to communicate and collaborate with team members via **Slack**.
 Once you have joined the **Slack Team** [EmpowerHack_LDN](http://slack.empowerhack.io), the channel for VaccinePass is `healthldn_vaccinepass`.

--- 

### Raw markdown for the preview above

```
## Slack Channel
 
This is the slack channel for those who would like to communicate and collaborate with team members via **Slack**.
 Once you have joined the **Slack Team** [EmpowerHack_LDN](http://slack.empowerhack.io), the channel for VaccinePass is `healthldn_vaccinepass`.
```